src/Query/QueryBuilder.php

```diff
@@ -105,8 +106,8 @@ class QueryBuilder
      *
      * In case you want to add custom conditions support, use the {@see setConditionClasses()} method.
      *
-     * @see setConditonClasses()
-     * @see defaultConditionClasses()
+     * {@see setConditonClasses()}
```

Comment:
No need for `{` and `}`. These are needed when you use it inside of text.

Reply:
done.